sci-electronics/iverilog/iverilog-10.3.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+GITHUB_PV=$(ver_rs 1- '_')
+
+DESCRIPTION="A Verilog simulation and synthesis tool"
+HOMEPAGE="http://iverilog.icarus.com"
```

Comment:
Could you add the Github page here also?

Reply:
> Could you add the Github page here also?

I've add it to ```HOMEPAGE``` array, thank you!

```
HOMEPAGE="
	http://iverilog.icarus.com
	https://github.com/steveicarus/iverilog
"
```